core/block_validator.go

```diff
@@ -407,7 +406,7 @@ func CalcGasLimit(parent *types.WorkObject, gasCeil uint64) uint64 {
 	var desiredLimit uint64
 	percentGasUsed := parent.GasUsed() * 100 / parent.GasLimit()
 	if percentGasUsed > params.PercentGasUsedThreshold {
-		desiredLimit = CalcGasCeil(parent.NumberU64(common.ZONE_CTX), gasCeil)
+		desiredLimit = gasCeil
 		if desiredLimit > gasCeil {
 			desiredLimit = gasCeil
```

Comment:
(nit) this logic seems strange to me

Reply:
previously we had different gas ceils at different block heights, but changed this to have one value